content/en/docs/setup/production-environment/container-runtimes.md

```diff
@@ -292,6 +302,10 @@ You should also note the changed `conmon_cgroup`, which has to be set to the val
 cgroup driver configuration of the kubelet (usually done via kubeadm) and CRI-O
 in sync.
 
+Starting with v1.28 and later, you can enable automatic detection of the
```

Comment:
```suggestion
Starting with Kubernetes v1.28 and later, you can enable automatic detection of the
```

Reply:
Fixed with `In Kubernetes {{< skew currentVersion >}}`